packages/artillery/lib/console-reporter.js

```diff
@@ -29,6 +29,7 @@ function ConsoleReporter(opts) {
   this.outputFormat = opts.outputFormat || process.env.OUTPUT_FORMAT || 'new';
 
   this.quiet = opts.quiet;
+  this.metricsToSuppress = opts.suppressMetrics;
```

Comment:
Nit but should probably just use the same name for external config and internally? (`metricsToSuppress` or `suppressMetrics`)